libobs/obs-output.h

```diff
@@ -77,6 +77,10 @@ struct obs_output_info {
 
 	/* raw audio callback for multi track outputs */
 	void (*raw_audio2)(void *data, size_t idx, struct audio_data *frames);
+
+	/* optional function to update obs_get_video and obs_get_audio ptrs */
+	void (*update_av)(void *context, struct obs_video *video,
+			  struct obs_sudio *audio);
```

Comment:
Neither 'obs_video or 'obs_sudio' is defined. From the use of this function, I assume that you meant to make these 'video_t' and 'audio_t' instead.